lib/utilities/reportHandler.js

```diff
@@ -96,38 +129,32 @@ function getCRRStats(log, cb) {
     const routes = backbeat.routes(redisKeys, sites);
     const details = routes.find(route =>
         route.category === 'metrics' && route.type === 'all');
-    // Add `site` as we're not using Backbeat's request parser for the API's URI
-    details.site = 'all';
-    return backbeatMetrics.getAllMetrics(details, (err, res) => {
+    return async.parallel({
+        total: done => _crrRequest(backbeatMetrics, details, 'all', done),
```

Comment:
This seems to break compatibility with the existing schema where backlog, completions and throughput are inline.